crates/rpc-proto/proto/account.proto

```diff
@@ -3,30 +3,48 @@ package account;
 
 import "digest.proto";
 
+// An account ID.
+//
+// A Miden account ID is a 120-bit value derived from the commitments to account code and storage,
+// and a random user-provided seed.
 message AccountId {
-    // A Miden account ID is a 120-bit value derived from the commitments to account code and
-    // storage, and a random user-provided seed.
+    // An account ID.
```

Comment:
We should also state the encoding i.e. how this should be interpreted into the two felts? Or is this somehow different to the prefix/suffix felt?

```suggestion
    // 15 bytes (120 bits) <..how to decode..>
```